examples/linux64/hello-make/Makefile

```diff
@@ -0,0 +1,15 @@
+CC = python -m ppci cc
+CFLAGS = -O1
```

Comment:
@windelbouwman: FYI, ppci-cc currently doesn't accept -O without arg, but it should (equivalent to -O1 per gcc).

Reply:
Hmm, indeed, this is a impedance mismatch with gcc. The way I implemented now is with argparse and accepting a series of optimization levels. The other option is to make each optimization target a seperate command line argument.

Reply:
> The way I implemented now is with argparse and accepting a series of optimization levels.

Yeah, I see the problem. argparse allows to specify an argument to short option either without a space or not, i.e. `ppci-cc -O1` and `ppci-cc -O 1` are equivalent, and `-O` alone is not possible, as argparse expects an argument. gcc "solves" that by now allow a space:
~~~
$ gcc -c -O 1 hello.c
gcc: error: 1: No such file or directory
~~~

> The other option is to make each optimization target a seperate command line argument.

That would make things like `-Os`, `-O1`, etc. essentially long options, and I thought argparse hardcodes long options to have `--` prefix. But trying:

~~~
+compile_parser.add_argument(
+    "-Ofoo", help="optimize code", action="store_true"
+)
~~~
And running:
~~~
`ppci-cc -Ofoo`
~~~

doesn't lead to exceptions or visible artifacts. Oh, it's Python, so checking that it actually works is a matter of seconds:

~~~
Namespace(E=False, I=[], M=False, O='0', Ofoo=True, S=False, ast=False, c=True, define=[], freestanding=False, g=False, html_report=None, include=[], instrument_functions=False, ir=False, log=20, machine=None, mtune=[], output='hello.o', pycode=False, report=None, sources=[<_io.TextIOWrapper name='hello.c' mode='r' encoding='UTF-8'>], std='c99', super_verbose=False, text_report=None, trigraphs=False, undefine=[], verbose=0, wasm=False)
~~~
So, yep, seems to work, and hopefully it means we'll be able to achieve as close option compatibility with gcc as possible without switching to custom arg parser.

Reply:
Okay, I suggest to fix this later, we'll figure a way :)